pmd-java/src/test/resources/net/sourceforge/pmd/lang/java/rule/design/xml/SimplifiedTernary.xml

```diff
@@ -53,12 +53,12 @@ public class Foo {
     </test-code>
 
     <test-code>
-        <description>condition ? true : false</description>
-        <expected-problems>0</expected-problems>
+        <description>#3603 condition ? true : false</description>
+        <expected-problems>1</expected-problems>
         <code><![CDATA[
 public class Foo {
     public boolean test() {
-        return condition ? true : false; // Existing rule
```

Comment:
> // Existing rule

I wonder, whether we have another rule that would pick up this return statement...

But overall I think you change makes sense - this test case was wrong before. It should flag this return statement